x-pack/plugins/reporting/server/export_types/csv_searchsource/generate_csv/generate_csv.ts

```diff
@@ -380,13 +396,16 @@ export class CsvGenerator {
       }
     }
 
-    this.logger.debug(`Finished generating. Row count: ${this.csvRowCount}.`);
+    this.logger.info(`Finished generating. Row count: ${this.csvRowCount}.`);
 
     if (!this.maxSizeReached && this.csvRowCount !== totalRecords) {
       this.logger.warn(
         `ES scroll returned fewer total hits than expected! ` +
           `Search result total hits: ${totalRecords}. Row count: ${this.csvRowCount}.`
       );
+      warnings.push(
```

Comment:
I don't have a reliable way of testing this scenario in a running environment, but we can see it working in one of the unit tests.